src/engine/bufferscalers/enginebufferscalerubberband.cpp

```diff
@@ -160,7 +160,7 @@ SINT EngineBufferScaleRubberBand::retrieveAndDeinterleave(
 
 void EngineBufferScaleRubberBand::deinterleaveAndProcess(
         const CSAMPLE* pBuffer, SINT frames, bool flush) {
-    DEBUG_ASSERT(frames <= static_cast<ssize_t>(m_buffers[0].size()));
+    DEBUG_ASSERT(frames <= static_cast<ptrdiff_t>(m_buffers[0].size()));
```

Comment:
It works, because SINT is ptrdiff_t; 
```suggestion
    DEBUG_ASSERT(frames <= static_cast<SINT>(m_buffers[0].size()));
```
Than it will work even if we change the definition of SINT one day.

Reply:
I changed it to `SINT` in cf664832c376fd2b35a1cf5ffdace8787ccee673.